geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/BoundingBox.scala

```diff
@@ -72,6 +72,13 @@ case class BoundingBox(ll: Point, ur: Point) {
    */
   def contains(otherGeom: Geometry): Boolean = geom.contains(otherGeom)
 
+  /**
+   * This bounding box equals bbox if the lower left and upper right points are the same.
+   * @param bbox
+   * @return
+   */
+  def equals(bbox: BoundingBox): Boolean = (ll == bbox.ll) && (ur == bbox.ur)
```

Comment:
The comparison of the points is logical and correct.  I just confused since there are a few notions of equality on the JVM.